internal/backend/db/dbgorm/scheme/embeddings.go

```diff
@@ -0,0 +1,17 @@
+// TODO: Make atlas/goose ignore these structs.
```

Comment:
1. The location of this comment makes it the "documentation" of this package
2. Add an issue ID

Reply:
removed comment. going to remove it manually from the migration per @efiShtain 's suggestion